flames/selenium/docker-compose.override.yml

```diff
@@ -0,0 +1,52 @@
+version: '2.1'
+
+services:
+
+    selenium-hub:
+        image: elgalu/selenium:${SELENIUM_SERVER:-latest}
+        hostname: ${HOST_NAME_SELENIUM:-selenium-hub}
+        ports:
+            - "${LOCAL_LISTEN_ADDR}${HOST_PORT_SELENIUM:-4444}:4444"
+        networks:
+            app_net:
+                ipv4_address: 172.16.238.150
+        volumes:
+            - /dev/shm:/dev/shm
```

Comment:
I suppose this is not going to work on Windows. AFAIK this should be handled via an argument. See here: https://stackoverflow.com/questions/24218960/docker-increase-size-of-the-dev-shm#24219049